Cargo.toml

```diff
@@ -33,6 +33,7 @@ secp256k1-zkp = { version = "0.10.1", features = ["rand", "rand-std", "global-co
 baid58 = "~0.4.4"
 mime = "~0.3.17"
 serde_crate = { package = "serde", version = "1", features = ["derive"], optional = true }
+serde_yaml = { version = "0.9.32", optional = true }
```

Comment:
Why do we need it here?

Reply:
We need it to be able to call `serde_yaml::with::singleton_map::deserialize`. The dependency was already in the graph so I though it was not a big issue to add it

Reply:
Oh, now I see it!